api/base/serializers.py

```diff
@@ -29,6 +29,65 @@ def _url_val(val, obj, serializer, **kwargs):
         return val
 
 
+class HyperLinkedIdentityFieldWithMeta(ser.HyperlinkedIdentityField):
+
+    def __init__(self, view_name=None, **kwargs):
```

Comment:
I meant specifically init. Can you call super on init and just add the count information as a single line?